app/ami.py

```diff
@@ -32,9 +32,8 @@ def to_header() -> [str]:
                 'AMI Type',
                 'OS',
                 'IOPS',
-                'VolumeType'
-                'Throughput',
-                'Snapshot IDs']
+                'VolumeType',
```

Comment:
I think for AMI we are missing the "AMI name" . Every AMI seems to have it, but I think we are pulling the Name tag. For example, https://us-west-1.console.aws.amazon.com/ec2/v2/home?region=us-west-1#Images:visibility=owned-by-me;search=ami-046531e848f15f786 doesn't show a name in the spreadsheet

Also, do we know if we have creation date available for all resources?